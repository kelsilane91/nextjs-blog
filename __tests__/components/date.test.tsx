import { render } from "@testing-library/react";
import Date from "../../components/date";
import "@testing-library/jest-dom";
import { expect } from "@jest/globals";

describe("Date", () => {
  const dateString = "2020-01-02";

  it("renders Date component unchanged", () => {
    const { container } = render(<Date dateString={dateString} />);
    expect(container).toMatchSnapshot();
  });
});
