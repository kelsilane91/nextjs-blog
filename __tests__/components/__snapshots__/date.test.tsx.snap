// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date renders Date component unchanged 1`] = `
<div>
  <time
    datetime="2020-01-02"
  >
    January 2, 2020
  </time>
</div>
`;
